spec/parliament/request/url_request_spec.rb

```diff
@@ -24,7 +25,7 @@
   end
 
   describe '#respond_to_missing?' do
-    subject { Parliament::Request::UrlRequest.new(base_url: 'http://test.com') }
+    subject { Parliament::Request::UrlRequest.new(base_url: 'http://test.com', decorators: Parliament::Grom::Decorator) }
```

Comment:
Looking at this, parliament-grom-decorators should be aded as a development dependency?